{
    "path": "java_projects/cleaned_final_projects_evosuite/commons-validator/src/test/java/org/apache/commons/validator/ValidatorException_ESTest.java",
    "imports": {
        "26-26:import EvoRunnerParameters": {
            "start": 26,
            "end": 26,
            "body": [
                "import org.evosuite.runtime.EvoRunnerParameters;\n"
            ]
        },
        "27-27:import RunWith": {
            "start": 27,
            "end": 27,
            "body": [
                "import org.junit.runner.RunWith;\n"
            ]
        },
        "25-25:import EvoRunner": {
            "start": 25,
            "end": 25,
            "body": [
                "import org.evosuite.runtime.EvoRunner;\n"
            ]
        },
        "22-22:import Test": {
            "start": 22,
            "end": 22,
            "body": [
                "import org.junit.Test;\n"
            ]
        },
        "24-24:import ValidatorException": {
            "start": 24,
            "end": 24,
            "body": [
                "import org.apache.commons.validator.ValidatorException;\n"
            ]
        },
        "23-23:import static Assert.*": {
            "start": 23,
            "end": 23,
            "body": [
                "import static org.junit.Assert.*;\n"
            ]
        }
    },
    "classes": {
        "ValidatorException_ESTest": {
            "start": 30,
            "end": 30,
            "is_abstract": false,
            "is_interface": false,
            "nested_inside": [],
            "nests": [],
            "implements": [],
            "extends": [
                "ValidatorException_ESTest_scaffolding"
            ],
            "methods": {
                "38-41:test1": {
                    "start": 38,
                    "end": 41,
                    "body": [
                        "  public void test1()  throws Throwable  {\n",
                        "      ValidatorException validatorException0 = ValidatorException.ValidatorException1();\n",
                        "      assertNotNull(validatorException0);\n",
                        "  }\n"
                    ],
                    "is_constructor": false,
                    "annotations": [
                        "Test(timeout = 4000)"
                    ],
                    "modifiers": [
                        "public"
                    ],
                    "return_types": [
                        [
                            "void",
                            "void"
                        ]
                    ],
                    "signature": "test1()",
                    "parameters": [],
                    "calls": [
                        [
                            "commons-validator.src.main.org.apache.commons.validator.ValidatorException",
                            "ValidatorException",
                            "34-36:ValidatorException1"
                        ],
                        [
                            "library",
                            "Assert",
                            "assertNotNull"
                        ]
                    ]
                },
                "33-35:test0": {
                    "start": 33,
                    "end": 35,
                    "body": [
                        "  public void test0()  throws Throwable  {\n",
                        "      ValidatorException validatorException0 = new ValidatorException(\"\");\n",
                        "  }\n"
                    ],
                    "is_constructor": false,
                    "annotations": [
                        "Test(timeout = 4000)"
                    ],
                    "modifiers": [
                        "public"
                    ],
                    "return_types": [
                        [
                            "void",
                            "void"
                        ]
                    ],
                    "signature": "test0()",
                    "parameters": [],
                    "calls": [
                        [
                            "commons-validator.src.main.org.apache.commons.validator.ValidatorException",
                            "ValidatorException",
                            "30-32:ValidatorException"
                        ]
                    ]
                }
            },
            "fields": {}
        }
    }
}